### PR TITLE
Bundle built assets

### PR DESCRIPTION
**Pull request by sambulance**
Branched from #30.

Commits built JS and CSS files to the project and enqueues them.

Todo: 
- [x] In dev mode, assets aren't enqueuing correctly.
- [x] Rename assets in prod mode to be more useful.
- [x] ~Only enqueue on pages where the feed is rendered.~ New issue: #36 
- [x] Update documentation.

### CODE REVIEW

**Comment by sambulance** [comment]
@jazzsequence Please can I have a preliminary review on this? Still a few bits to iron out (listed above) but for the most part this is doing the job.

File: inc/load-more.php [hunk 1]
```diff
@@ -69,14 +69,32 @@ function prepare_response() {
...
-	Asset_Loader\autoregister( dirname( __DIR__ ) . '/build', 'hm-juicer-load-more', [
-		'handle'  => 'hm-juicer-load-more',
-		'scripts' => [ 'jquery', 'underscore', 'hm-juicer-js' ],
-	] );
-
-	// Enqueue custom CSS for the HM Juicer layout.
-	Asset_Loader\autoenqueue( dirname( __DIR__ ) . '/build', 'hm-juicer-style', [
-		'handle'    => 'hm-juicer-style',
-	] );
+	$handle = 'hm-juicer-load-more';
```
Comment (jazzsequence) [inline inc/load-more.php]: align `=`
🔢

File: inc/load-more.php [hunk 2]
```diff
@@ -69,14 +69,32 @@ function prepare_response() {
...
+		/**
+		 * Production mode. Use standard WordPress enqueueing for built assets.
+		 */
+
+		// JS.
+		wp_enqueue_script(
+			$handle,
+			plugins_url( '/build/prod/load_more.js', dirname( __FILE__ ) ),
+			$dependencies,
+			'0.0.1', // TODO: use plugin version.
```
Comment (jazzsequence) [inline inc/load-more.php]: FWIW the first release will be `0.1.0` if you want to just hard code that.

File: package.json [hunk 3]
```diff
@@ -23,7 +23,7 @@
         "last 2 Opera versions"
     ],
     "scripts": {
-        "build": "webpack --config .config/wepback-config-prod.js",
+        "build": "webpack --config .config/webpack-config-prod.js",
```
Comment (jazzsequence) [inline package.json]: 🤦‍♂

**Comment by jazzsequence** [comment]
> In dev mode, assets aren't enqueuing correctly.

The webpack dev config may need to be updated to compile in the new path. But that's a shot in the dark since I don't know what the specific issues are.

**Pull request merged**